### DISCUSSION TITLE
Addon for Opera GX?

### DISCUSSION THREAD

**Discussion by Duckie360** [ScratchAddons/ScratchAddons#4547] [Help] [answered]
I use Opera GX as my main browser. Could it be added onto the Opera store? I have the ZIP folder on my computer but I don't know what to do with it. Would it work if I put it into Opera's files?

edit: sorry for the inconvenience, I figured out how to by using the Chrome version.

**Comment by Norbiros** [comment, marked as answer]
Probably you can't, but opera created this extension:
https://addons.opera.com/en/extensions/details/install-chrome-extensions/
So you can use all of your chrome's extensions on opera GX
